lib/rdoc/parser/c.rb

```diff
@@ -1102,35 +1093,10 @@ def load_variable_map(map_name)
   # Both :main: and :title: directives are deprecated and will be removed in RDoc 7.
 
   def look_for_directives_in(context, comment)
-    @preprocess.handle comment, context do |directive, param|
-      case directive
-      when 'main' then
```

Comment:
I still want to print these deprecation warnings until we drop them in v7 release